webpack/assets/javascripts/react_app/components/LoginPage/LoginPage.fixtures.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+export const adjustProps = ({ token, version }) => ({
+  container: {
+    backgroundUrl: '/assets/login_background.jpg',
+    alert: {
+      message: null,
+      onDismiss: null,
+      show: false,
+    },
+  },
+  header: {
+    logoSrc: '/assets/login_logo.png',
+    caption:
+        <React.Fragment>
+            <h1>Welcome</h1>
+            <p>{__(`Version ${version.version}`)}</p>
+        </React.Fragment>,
+  },
+  card: {
+    title: 'Log into your account',
+    form: {
+      attributes: {
+        action: '/users/login',
+        method: 'post',
+      },
+      validate: false,
+      submitError: null,
+      showError: false,
+      usernameField: {
+        id: 'login[login]',
+        attributes: {
+          name: 'login[login]',
+        },
+        type: 'email',
+        placeholder: 'Email address',
+        errors: {},
+        error: null,
+        showError: false,
+      },
+      passwordField: {
+        id: 'login[password]',
+        attributes: {
+          name: 'login[password]',
+        },
+        type: 'password',
+        placeholder: 'Password',
+        minLength: 8,
+        errors: {},
+        warnings: {},
+        warning: null,
+        showWarning: false,
+      },
+      additionalFields: <input name="authenticity_token" type="hidden" value={token} />,
+      disableSubmit: false,
+      submitText: 'Log In',
+      onSubmit: ({ target: form }) => form.submit(),
```

Comment:
how do you show validation errors (e.g. bad login?)

Reply:
still a WIP but tomorrow I'll start work on that.